core/src/main/java/com/imaginea/kodebeagle/GitHelper/Constants.java

```diff
@@ -0,0 +1,490 @@
+package com.imaginea.kodebeagle.GitHelper;

```

Comment:
Follow the same naming convention for the packages as is being used currently. Use "com.kodebeagle" as the root.